### DISCUSSION TITLE
the 'follow' button is always on top even you click the 'menu' (mobile webpages)

### DISCUSSION THREAD

**Discussion by LynnLvC** [mmistakes/minimal-mistakes#4602] [General]
As mentioned in the title, and see the attached figure. 
It only appear in responsive page because only then the follow button will appear.
I tried to tune the index but does not work.
Thanks for any help.

![Screenshot 2023-12-14 114256](https://github.com/mmistakes/minimal-mistakes/assets/9216429/b08e3915-6eca-48b1-9889-6bf94965abb5)

**Comment by thanhag** [comment]
I was about to open a discussion to ask the community for guidance on how to fix it, but I saw you had the same problem.
I don't know in which file I should fix this "Follow" problem. If anyone knows, please guide
![be746ba11100b95ee011](https://github.com/mmistakes/minimal-mistakes/assets/154648990/7d155d74-f394-4fee-be85-ca179e86821f)

**Comment by thanhag** [comment]
I checked with DevTool on Chrome (and Brave), installed all smartphones, and all did not have this error. The picture below shows it working ok.

![image](https://github.com/mmistakes/minimal-mistakes/assets/154648990/4e40894b-0c92-49dd-b447-a8ce9dbc5382)

Only when loading the website on my actual Android device does the above error appear.
Does anyone have any ideas, I don't really have much scss and javascript experience yet
